topics/variant-analysis/tutorials/MTB_transmission/tutorial.md

```diff
@@ -0,0 +1,869 @@
+---
+layout: tutorial_hands_on
+
+title: 'Identifying tuberculosis transmission links: from SNPs to transmission clusters'
+zenodo_link: ''
+objectives:
+- Create a SNP alignment
+- Calculate pairwise SNP distances between MTB samples
+- Identify transmission clusters based on SNP distances
+- Study the emergence and spread of drug resistance based on transmission analysis.
+time_estimation: 2H
+enable: true
+level : Intermediate
+key_points:
+- Clustering is a useful tool to detect transmission links between patients and oubreak investigation.
+- Clustering can be used to investigate the transmission of certain traits, like drug resistance.
+- Clustering does not provide information about particular transmission events nor their directionality (who infected whom).
+- Clustering is very much influenced by sampling. Lower sampling proportions and shorter sampling timeframes lead to lower clustering rates that shoud not be confounded with lack of transmission.
+contributors:
+- Galo A. Goig
+- Daniela Brites
+- Christoph Stritt
```

Comment:
These three people need to be added to the `CONTRIBUTORS.yaml` file, like you did in https://github.com/galaxyproject/training-material/pull/3210, and then you should use their 'username' rather than their human name, to reference the entry in the CONTRIBUTORS.yaml file.

Reply:
Thank you, I will do it now

Reply:
Just one thing that is not clear to me: I added the contributors and commited the change to the main branch in my forked repository. Is this automatically picked up by this pull request or do I need to create a new one?

Reply:
Yes, it is. This looks good. The only thing that you can improve next time is, to not commit anything to your main branch. Always use a new branch :)